tools/ci_connector_ops/ci_connector_ops/pipelines/contexts.py

```diff
@@ -312,8 +303,6 @@ def __init__(
             gha_workflow_run_url=gha_workflow_run_url,
             pipeline_start_timestamp=pipeline_start_timestamp,
             ci_context=ci_context,
-            # TODO: remove this once stable and our default pipeline
-            is_ci_optional=True,
```

Comment:
@bnchrch I'm not sure why you marked all the connector specific status check as optional. If they fail developers should still be allowed to merge their PR as these checks are not required yet.

Reply:
Discussed in our call yesterday! Totally fine to remove